lib/filesystem/CArchiveLoader.h

```diff
@@ -68,9 +68,9 @@ class DLL_LINKAGE CArchiveLoader : public ISimpleResourceLoader
 	void updateFilteredFiles(std::function<bool(const std::string &)> filter) const override {}
 	std::unordered_set<ResourceID> getFilteredFiles(std::function<bool(const ResourceID &)> filter) const override;
 	/** Extracts one archive entry to the specified subfolder. Used for Video and Sound */
-	void extractToFolder(const std::string & outputSubFolder, CInputStream & fileStream, ArchiveEntry entry);
+	static void extractToFolder(const std::string & outputSubFolder, CInputStream & fileStream, const ArchiveEntry & entry);
```

Comment:
as far as I remember there was an idea to not make methods static, especially if there is non-static overload. The idea is that this method still describes instance behavior and may need some object state later, during some refactoring or whatever.

Reply:
That's what I keep telling whenever I see addition of `static`.
User code should make no assumptions as to whether method accesses class members in call.

Missed this case probably

Reply:
removed static